docs/en/ingest-management/fleet/set-inactivity-timeout.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[set-inactivity-timeout]]
+= Set inactivity timeout
+
+[[why-set-inactivity-timeout]]
+== Why set an inactivity timeout?
+
+The inactivity timeout moves {agent}s to inactive status after being offline for a set amount of time. 
+Inactive {agent}s are still valid {agent}s, but are removed from the main {fleet} UI allowing you to better manage {agent}s and 
+declutter the {fleet} UI.
+
+Once {fleet-server} receives a check-in from an inactive {agent}, it returns to healthy status. 
+
+For example, if an employee is on holiday with their laptop off, 
+the {agent} will transition to offline then inactive once the inactivity timeout limit is reached. 
+This prevents the inactive {agent} from cluttering the {fleet} UI.
+Once the employee returns, the {agent} checks in and returns to healthy status with valid API keys. 

```

Comment:
```suggestion
When {fleet-server} receives a check-in from an inactive {agent}, it returns to healthy status. 

For example, if an employee is on holiday with their laptop off, 
the {agent} will transition to offline then inactive when the inactivity timeout limit is reached. 
This prevents the inactive {agent} from cluttering the {fleet} UI.
When the employee returns, the {agent} checks in and returns to healthy status with valid API keys. 

```
Changing once > when because I've always been told by editors to avoid "once" because it might cause ambiguity during translation (not that we translate, but....). After or when are preferred.